vms/platformvm/service.go

```diff
@@ -2615,6 +2615,7 @@ func (s *Service) getAPIOwner(owner *secp256k1fx.OutputOwners) (*platformapi.Own
 	apiOwner := &platformapi.Owner{
 		Locktime:  json.Uint64(owner.Locktime),
 		Threshold: json.Uint32(owner.Threshold),
+		Addresses: make([]string, 0, len(owner.Addrs)),
```

Comment:
we should modify `apiOwner.Addresses = append(apiOwner.Addresses, addrStr)` to take advantage of this preallocation